pycbc/conversions.py

```diff
@@ -626,6 +626,94 @@ def _det_tc(detector_name, ra, dec, tc, ref_frame='geocentric'):
 
 det_tc = numpy.vectorize(_det_tc)
 
+def _detector_to_optimal_orientation(detector_name, tc):
+    """ Low-level function to be called from _optimal_dec_from_detector
+    and _optimal_ra_from_detector"""
+
+    d = Detector(detector_name)
+    ra, dec = d.optimal_orientation(tc)
+
+    return ra, dec
+
+
+def _optimal_dec_from_detector(detector_name, tc):
+    """For a given detector and GPS time, return the optimal orientation
+    in declination.
+
+    A typical use case: Given a detector D and GPS time T, one wants to
+    inject signals that are optimally oriented. Therefore, values of
+    right ascension (ra) and declination (dec) are not known before calling
+    pycbc_create_injections. This transform computes them on-the-fly.
+
+    Example:
+
+    >>> pycbc_create_injections --config-files params.ini
+    >>> cat params.ini
+
+    [waveform_transforms-ra+dec]
+    name = custom
+    inputs = tc
+    ra = optimal_ra_from_detector("H1",tc)
+    dec = optimal_dec_from_detector("H1",tc)
```

Comment:
Drop the example text, since that about how to run `pycbc_create_injections` (there is documentation about that in that program's help). Instead, give a definition of what "optimal" means: directly over head of the detector.